src/client.ts

```diff
@@ -1591,6 +1593,30 @@ export class StreamChat<
     });
   }
 
+  /**
+   * getRateLimits - Returns the rate limits quota and usage for the current app, possibly filter for a specific platform and/or endpoints.
+   * Only available server-side.
+   *
+   * @param {object} [params] The params for the call. If none of the params are set, all limits for all platforms are returned.
+   * @returns {APIResponse & GetRateLimitsResponse}
+   */
+  async getRateLimits(params?: {
+    android?: boolean;
+    endpoints?: EndpointName[];
+    ios?: boolean;
+    serverSide?: boolean;
+    web?: boolean;
+  }) {
+    const { serverSide, web, android, ios, endpoints } = params || {};
+    return this.get<APIResponse & GetRateLimitsResponse>(this.baseURL + '/rate_limits', {
```

Comment:
Can we create a separate type for this?

`export type GetRateLimitsApiResponse = APIResponse & GetRateLimitsResponse`

Reply:
https://github.com/GetStream/stream-chat-js/pull/631/commits/8b02e222816d9ffbaddccb5ab7c1a0a2920bbece